Testbed/CMakeLists.txt

```diff
@@ -48,25 +52,43 @@ if(APPLE)
 elseif(WIN32)
 	set (ADDITIONAL_LIBRARIES winmm)
 endif(APPLE)
-
-add_executable(Testbed
-	${Testbed_Framework_SRCS}
-	${Testbed_Tests_SRCS}
-)
 
-#Resolve Linker error LNK4098; make sure default libcmt doesn't clash with other libraries
+# Resolve Linker error LNK4098; make sure default libcmt doesn't clash with other libraries
 if(MSVC)
 	set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /NODEFAULTLIB:libcmt.lib")
 endif()
 
-target_link_libraries (
-	Testbed
-	PlayRho
-	${GLEW_LIBRARY}
-	${GLFW_STATIC_LIBRARIES}
-	${ADDITIONAL_LIBRARIES}
-	${OPENGL_LIBRARIES}
-)
+add_executable(Testbed
+	${Testbed_Framework_SRCS}
+	${Testbed_Tests_SRCS}
+)
+
+include_directories(
+	${OPENGL_INCLUDE_DIR}
+	${GLEW_INCLUDE_DIRS}
+	${GLFW_INCLUDE_DIRS}
+	${PlayRho_SOURCE_DIR}
+)
+
+if(GLFW_TARGET)
+	target_link_libraries(
+		Testbed
+		PlayRho
+		glfw
```

Comment:
I'm new to pull request reviews as well. But I've since reverted the commit, and re-applied it. So the `View changes` button no longer takes me anywhere useful.